ister_gui.py

```diff
@@ -1423,6 +1423,7 @@ def _static_configuration(self, _):
         try:
             subprocess.call(['/usr/bin/systemctl', 'restart',
                              'systemd-networkd', 'systemd-resolved'])
+            time.sleep(.1)
```

Comment:
So could we possibly poll (look up ip a output or something) instead of sleeping here? There is another place or two with sleeps that it would be nice to avoid if possible (especially where if we don't wait long enough, there is a problem).